chore(docs): improve community documentation (#1066)
hashicorp/terraform-cdk: website/docs/cdktf/community.html.md

# Community

> CDK for Terraform is a rapidly-growing project and the development team would love your feedback and direct contributions to help guide the project.

Reviewer: ```suggestion
CDK for Terraform is growing rapidly, and the development team would love your feedback and direct contributions to help guide the project.
```

Reviewer 2: Rather than "growing rapidly" or "early experimental" maybe "CDK for Terraform is under active development" is a better way to phrase?

Reviewer: okay! updated suggestion to reflect!

## Questions & Issues

> If you have questions, the HashiCorp [Discuss](https://discuss.hashicorp.com/) platform using the [terraform-cdk](https://discuss.hashicorp.com/c/terraform-core/cdk-for-terraform/) category is the best place for general discussion. If you run into issues or have ideas for improvements, please file a [bug](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=bug&template=bug-report.md&title=) or request a new [feature](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=enhancement&template=feature-request.md&title=). Please let us know if you'd like to talk - we are very interested in seeing how people use CDK for Terraform in practice.

Reviewer: ```suggestion
If you have questions, the HashiCorp [Discuss](https://discuss.hashicorp.com/) platform using the [terraform-cdk](https://discuss.hashicorp.com/c/terraform-core/cdk-for-terraform/) category is the best place for general discussion. If you run into issues or have ideas for improvements, please file a [bug](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=bug&template=bug-report.md&title=) or request a new [feature](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=enhancement&template=feature-request.md&title=). Please let us know if you'd like to talk - we are very interested in learning how people use CDK for Terraform in practice.
```

## Contributing

> The CDK for Terraform project is very open to community contributions. The easiest way to help is to file bugs when you run into issues, comment on existing issues if you're seeing them, and comment or upvote on feature requests if you have use cases where they would be helpful. Hearing _why_ you're interested extremely useful.

Reviewer: ```suggestion
The CDK for Terraform project is very open to community contributions. The easiest ways to help are by filing bugs for issues, commenting on existing issues you encounter, and commenting on or upvoting feature requests when you have use cases where they would be helpful. Hearing _why_ you're interested is extremely useful, and we appreciate your feedback.
```

> If you're interested in contributing code, please look at the [CONTRIBUTING.md](./CONTRIBUTING.md) guide. Before you file a large pull request, please file an enhancement request, and tell us that you're interested in building it. We're happy to work together on design, to make sure your PR can be reviewed and accepted quickly.

Reviewer: Is there also a way to provide a link to an enhancement request? 

```suggestion
If you're interested in contributing code, please look at the [CONTRIBUTING.md](./CONTRIBUTING.md) guide. Before you file a large pull request, please file an enhancement request to tell us what you're interested in building. We're happy to work with you on the design to make sure we can review and accept your PR quickly.
```

Reviewer 2: @laurapacilio this is linked in the # Questions and Issues section above, so maybe just a question of where it fits best?

> request a new [feature](https://github.com/hashicorp/terraform-cdk/issues/new?assignees=&labels=enhancement&template=feature-request.md&title=)

Reviewer: oh! I thought reading this that enhancement was different somehow - maybe we just say "request a new feature"? We don't need to link twice...I just think the use of enhancement confused me

Reviewer 2: enhancement = new feature :) so the same issue template would apply. I'm flexible on the wording

Reviewer: okay! I updated my comment to use feature instead - thanks!

## Roadmap

> CDK for Terraform is released on a monthly cadence. To see what's planned next, look at [GitHub milestones](https://github.com/hashicorp/terraform-cdk/milestones). Typically, releases beyond the next are planned tentatively. Issues labeled `committed` will land in the planned release, and all others are implicitly stretch goals, and may shift to a future release if they don't get done in time for the release. 
> 

Reviewer: ```suggestion
CDK for Terraform is released on a monthly cadence. The [GitHub milestones] (https://github.com/hashicorp/terraform-cdk/milestones) for the project show what is planned for the next release and tentatively planned for subsequent releases. Issues labeled `committed` will land in the planned release, and all others are implicitly stretch goals that may shift to a future release as needed. 
```
